packages/python-packages/doc-warden/warden/WardenConfiguration.py

```diff
@@ -42,7 +42,13 @@ def __init__(self):
             '-d',
             '--scan-directory',
             dest = 'scan_directory',
-            help = 'The repo directory that this tool should be scanning.',
+            help = 'The directory on a repo that this tool should be scanning.',
+            required = True)
+        parser.add_argument(
+            '-h',
+            '--repo-root',
```

Comment:
Do we use target_directory for service directory and repo_root for entire repo?

Can we parse repo_root from target_directory instead of taking another arg?

Or having some default logic which make it not required.

Reply:
That could work.